.changes/tauri-app-in-same-dir.md

```diff
@@ -0,0 +1,6 @@
+---
+'tauri-cli': 'patch:feat'
+'@tauri-apps/cli': 'patch:feat'
+---
+
+Support Tauri configuration (`tauri.conf.json(5)` / `Tauri.toml`) and `package.json` to be in the same directory.
```

Comment:
I couldn't produce the issue mentioned here so I will still accept the PR as it contains an improvement anyway